### PR TITLE
Create stand-cheat-sheet.adoc

### PR DESCRIPTION
**Pull request by rtyler**
This is still a work in progress, see [EVENTS-26](https://issues.jenkins-ci.org/browse/EVENTS-26)

### CODE REVIEW

**Comment by rtyler** [comment]
I'd like @orrc, @fredg02 and/or @alyssat to take a look at this before I call it any form of "ready" and print them off

File: fosdem/stand-cheatsheet.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+The pins with flags on them are for contributors and JAM organizers.
+
+=== Postcard
+
+The postcard printed should be given to everybody that receives or takes a sticker and/or pin.
+
+On the front of the postcard are quick links to "solution pages" which contain links to plugins, articles, presentations and other documentation for using Jenkins on that platform/toolchain.
+
+On the back of the postcard is a current map for regional Jenkins Area Meetups, a live version of this map can be shown/found at link:https://meetup.com/pro/jenkins[meetup.com/pro/jenkins].
```
Comment (daniel-beck) [inline fosdem/stand-cheatsheet.adoc]: Link doesn't work for me.

File: fosdem/stand-cheatsheet.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+=== Stickers
+
+We have a number of different kinds of stickers, the names are:
+
+* Jenkins (traditional logo)
+* Angry Jenkins (fiery red logo)
+* Ninja Jenkins
+* General Jenkins (Jenkins in a military outfit)
+* Cute Jenkins (Jenkins in make a Anime-style "whoops" face)
```
Comment (daniel-beck) [inline fosdem/stand-cheatsheet.adoc]: Grammar?

File: fosdem/stand-cheatsheet.adoc (hunk 1 above)
Comment (rtyler) [inline fosdem/stand-cheatsheet.adoc]: fixed, stupid www vs. not

File: fosdem/stand-cheatsheet.adoc (hunk 2 above)
Comment (rtyler) [inline fosdem/stand-cheatsheet.adoc]: corrected

**Comment by fredg02** [comment]
:+1:

File: fosdem/stand-cheatsheet.adoc [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+=== Available Demos
+
+There are a number of Docker containers available and pre-configured with
+demonstration jobs on the laptop. Each of these should be linked from within
+the Chrome browser's bookmarks bar.
+
+* *PHP:* Using an example from link:http://jenkins-php.org[jenkins-php.org]
+* *Pipeline:* A couple of example jobs using the link:https://wiki.jenkins-ci.org/display/JENKINS/Workflow+Plugin[Pipeline plugin] to implement a few projects
+* *Python:* A couple examples jobs using the plugins suggested on link:https://jenkins-ci.org/s/python[jenkins-ci.org/s/python]
```
Comment (orrc) [inline fosdem/stand-cheatsheet.adoc]: "A couple examples jobs" -> "A couple of example jobs"

File: fosdem/stand-cheatsheet.adoc [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+The postcard printed should be given to everybody that receives or takes a
+sticker and/or pin.
+
+On the front of the postcard are quick links to "solution pages" which contain
+links to plugins, articles, presentations and other documentation for using
+Jenkins on that platform/toolchain.
+
+On the back of the postcard is a current map for regional Jenkins Area
+Meetups,a live version of this map can be shown/found at
```
Comment (orrc) [inline fosdem/stand-cheatsheet.adoc]: "Meetups,a live version" -> "Meetups; a live version"

File: fosdem/stand-cheatsheet.adoc [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+link:https://www.meetup.com/pro/jenkins[meetup.com/pro/jenkins].
+
+
+== Merchandise
+
+=== T-Shirts
+
+T-shirts are *not free* but are to be sold, all proceeds go directly to the
```
Comment (orrc) [inline fosdem/stand-cheatsheet.adoc:100]: Do we know the price they should be sold for?  I guess we can print / write that on separate signs anyway.

**Comment by orrc** [comment]
:+1:

File: fosdem/stand-cheatsheet.adoc (hunk 5 above)
Comment (rtyler) [inline fosdem/stand-cheatsheet.adoc:100]: I'm waiting for @alyssat to get online after her international trip to answer that question, I actually don't know but will update before I print off some of these sheets

**Pull request merged**